modules/Bio/EnsEMBL/Compara/PipeConfig/HomologyAnnotation_conf.pm

```diff
@@ -81,8 +81,8 @@ sub default_options {
         'decision_capacity'        => 150,
         'hc_priority'              => -10,
 
-        'num_sequences_per_blast_job'   => 50,
-        'all_blast_params'              => [ 35, 50, '-seg no -max_hsps 1 -use_sw_tback -num_threads 1 -matrix PAM70 -word_size 2', '1e-6' ],
+        'num_sequences_per_blast_job'   => 200,
+        'all_blast_params'              => [ 35, 50, '--max-hsps 1 --matrix PAM70 --threads 4 -b1 -c1 --sensitive', '1e-6' ],
```

Comment:
Yes, I simplified what we had with the expectation that all and any parameters will change, until we know what we really want from the homology annotation pipeline, what type of clustering we plan to incorporate, the parameters at this point are a simple default! Also, given later clustering, to have different matrix scoring, this would skew any sort of RBH analysis we seem to be looking to perform. Processing the proteins based on length of sequence should come later on in the processing of the `peptide_align_feature` table. I have got rid of the matrix specification at all now actually and reverted as you have suggested, because I missed that I had kept that part in, the default used by `BLASTp` and `Diamond blastp` is `BLOSUM62` - which works for distantly related species as well as more closely - given we are going to have a **set** reference database. I think this is something we will need to consider later on when processing the PAF table though.

Reply:
FYI we introduced the per-size blast parameters after realising that alignments of short proteins could not reach the minimum evalue simply even if the alignment was good because they were too short. See ENSCOMPARASW-291 for the full story.
I think you're right: another way of looking at that is that we decided to keep a constant e-value cutoff, so had to fix the generation of the e-values, whereas here we can make the clustering adapt itself according to the length of the proteins, apply a variable cutoff, etc. That said, you would still need to have a permissive cutoff here so that the hits get recorded in the database. 1e-6 sounds good enough, but we can double check the coverage on small proteins later on with the query that is in the ticket